a10_octavia/controller/worker/flows/vthunder_flows.py

```diff
@@ -53,12 +54,11 @@ def get_create_vthunder_flow(self):
         :returns: The flow for creating the amphora
         """
 
-        create_vthunder_flow = linear_flow.Flow(constants.CREATE_AMPHORA_FLOW)
+        create_vthunder_flow = linear_flow.Flow(
+            constants.CREATE_AMPHORA_FLOW)
         sf_name = 'spare_vthunder_create'
         create_vthunder_flow.add(database_tasks.CreateAmphoraInDB(
-            name=sf_name + '-' + constants.CREATE_AMPHORA_INDB,
-            provides=constants.AMPHORA_ID))
-
+            name=sf_name + '-' + constants.CREATE_AMPHORA_INDB))
```

Comment:
seems like code is overwritten here ? I dont see "provides" line which provides "constants.AMPHORA_ID" which is referenced in next code